crypto/bio/socket.cc

```
@@ -123,14 +123,14 @@ static const BIO_METHOD methods_sockp = {
     nullptr /* callback_ctrl */,
 };
 
-const BIO_METHOD *BIO_s_socket(void) { return &methods_sockp; }
+const BIO_METHOD *BIO_s_socket() { return &methods_sockp; }
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)